docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,5 +7,11 @@ Add a summary of the change and which issue is fixed, including motivation, cont
 - [ ] Code follows the style guidelines
 - [ ] Code commented in relevant places
 - [ ] Changes to necessary documentation made
-- [ ] New and existing unit tests pass locally with changes
```

Comment:
You might want to retain this line because in my experience people forget to run existing 'automated' unit tests and it might break. Since you guys dont have jenkins or something it is useful to keep this bullet.